web-common/src/features/entity-management/actions.ts

```diff
@@ -40,6 +49,19 @@ export async function renameFileArtifact(
     });
 
     httpRequestQueue.removeByName(fromName);
+
+    const [fromFolder] = splitFolderAndName(fromPath);
+    const [toFolder] = splitFolderAndName(toPath);
+    if (
+      fromResName?.kind &&
+      FolderToResourceKind[removeLeadingSlash(fromFolder)] ===
+        fromResName.kind &&
+      fromFolder !== toFolder
+    ) {
+      notifications.send({
+        message: `Moving ${fromName} out of native folder. Please make sure to add "kind" key to denote the type.`,
+      });
+    }
```

Comment:
This notification is firing even when moving a file into a subdirectory, say from `models/abc.sql` to `models/sub-dir/abc.sql`. We don't need to manually add a `kind` in subdirectories, right?

Reply:
Good point. It only needs the root folder to be `models`,`sources` etc